query/README.md

```diff
@@ -0,0 +1,42 @@
+# Query FHIR Data and Views
+This directory contains our new (under development) approach for querying
+transformed FHIR data which is replacing the [old approach](../dwh). The new
```

Comment:
Do we need to reference the old approach? We'd have to remember to update this when we get rid of `dwh`
```suggestion
transformed FHIR data. The new
```